examples/barotropicqg_betaforced.jl

```diff
@@ -211,27 +231,36 @@ nothing # hide
 
 startwalltime = time()
 
-while cl.step < nsteps
-  stepforward!(prob, diags, nsubs)
-
-  BarotropicQG.updatevars!(prob)
-  cfl = cl.dt*maximum([maximum(v.u)/g.dx, maximum(v.v)/g.dy])
+p = plot_output(prob)
 
+anim = @animate for j=0:Int(nsteps/nsubs)
+        
+  cfl = cl.dt*maximum([maximum(vs.u)/gr.dx, maximum(vs.v)/gr.dy])
+  
   log = @sprintf("step: %04d, t: %d, cfl: %.2f, E: %.4f, Q: %.4f, walltime: %.2f min",
-    cl.step, cl.t, cfl, E.data[E.i], Z.data[Z.i], 
-    (time()-startwalltime)/60)
-  println(log)
+  cl.step, cl.t, cfl, E.data[E.i], Z.data[Z.i], 
+  (time()-startwalltime)/60)
+  
+  if j%(1000/nsubs)==0; println(log) end  
+  
+  p[1][1][:z] = Array(vs.zeta)
```

Comment:
I suppose the call to `Array` is to ensure this works on the GPU? Could be good to add a comment here, especially since this is an example.

Reply:
This is **exactly** the problem I wanted you to help me out.
It's a bit complicated... if I don't us `Array()` then `@animate` does not use the first frame for vorticity. Very strange....